tests/src/texture/hipBindTexObj1D.cpp

```diff
@@ -0,0 +1,97 @@
+/* HIT_START
+ * BUILD: %t %s ../test_common.cpp EXCLUDE_HIP_PLATFORM nvcc
+ * RUN: %t
+ * HIT_END
+ */
+
+#include<iostream>
+#include<string.h>
+#include"hip/hip_runtime.h"
+#include"hip/hip_runtime_api.h"
+#include "test_common.h"
+
+#define N 512
+using namespace std;
+
+texture<int, hipTextureType1D,hipReadModeElementType> tex;
+
+bool testResult=true;
+
+__global__ void kernel(int *out)
+{
+
+	int x=blockIdx.x * blockDim.x + threadIdx.x;
+	out[x]=tex1Dfetch(tex,x);
+
+}
+
+
+
+void runTest(void);
+
+int main(int argc, char** argv) {
+    runTest();
+
+    if (testResult) {
+        passed();
+    } else {
+        exit(EXIT_FAILURE);
+    }
+}
+
+void runTest()
+
+{
+string out;
```

Comment:
Formatting is all wrong. Can you please run ```clang-format --style=file``` on this file? You can either use the clang-format in HCC bin directory or one that comes with clang/llvm.